js/menus/menuTemplate.js

```diff
@@ -184,17 +186,35 @@ function getTemplate(app) {
             checked: launchOnStartup,
             click: function (item) {
                 if (item.checked){
-                    symphonyAutoLauncher.enable()
-                    .catch(function (err) {
-                        let title = 'Error setting AutoLaunch configuration';
-                        electron.dialog.showErrorBox(title, title + ': ' + err);
-                    });
+                    if (isMac){
+                        childProcess.exec(`launchctl load ${launchAgentPath}`, (err) => {
```

Comment:
As informed earlier in this ticket there are open issue for MacOS, we've built a custom solution. Please check the ticket here -> [issue](https://github.com/Teamwork/node-auto-launch/issues/28)